docs/index.md

```diff
@@ -63,7 +63,7 @@ The Artifactory provider supports two ways of authentication. The following meth
 
 ### Access Token
 Artifactory access tokens may be used via the Authorization header by providing the `access_token` field to the provider
-block. Getting this value from the environment is supported with the `ARTIFACTORY_ACCESS_TOKEN` variable
+block. Getting this value from the environment is supported with `JFROG_ACCESS_TOKEN` or `ARTIFACTORY_ACCESS_TOKEN` variables
```

Comment:
I feel like it's important to mention what the intent is. I'm kind of directing this to jfrog team